src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -138,32 +141,37 @@ void ColorScheme::LayerJson(const Json::Value& json)
     }
 }
 
-std::wstring_view ColorScheme::GetName() const noexcept
+winrt::hstring ColorScheme::Name() const noexcept
 {
-    return { _schemeName };
+    return { _schemeName.c_str() };
 }
 
-std::array<til::color, COLOR_TABLE_SIZE>& ColorScheme::GetTable() noexcept
+winrt::com_array<Color> ColorScheme::Table() noexcept

```

Comment:
it is critically important that we validate the users of this function.
The old version _let recipients edit the color table_
This version _does not_

Reply:
Looks like every recipient only reads these values. So that's good. I'll add a const to the function header to make that clear though.

The closest I found was that `TerminalSettings` has a `SetColorTableEntry()`, but it makes sense that we don't need that (right now, at least) because we're only reading the settings, not writing.